redisinsight/ui/src/components/base/theme/types.ts

```diff
@@ -7,3 +7,11 @@ export type CommonProps = {
 }
 
 export type Theme = ReturnType<typeof useTheme>
+
+const ASC = 'asc'
+const DESC = 'desc'
+export type Direction = typeof ASC | typeof DESC
```

Comment:
nit: I've already mentioned this in other [comment](https://github.com/redis/RedisInsight/pull/4841#discussion_r2297271411), but this is a great example in which using enum will make it simpler. 

And yes, it's a subjective statement, I'm aware of that "more readable" is not a strong argument 😀

```
export enum Direction {
  Asc = 'asc',
  Desc = 'desc'
}
```